Automattic/course
# Style Course Theme buttons

This relates to #13 

The Design : 
<img width="964" alt="image" src="https://user-images.githubusercontent.com/7208249/195332909-0db8afef-eebc-441c-a819-b8faf3e4a0cf.png">

How to test: 
1. Create a new page 
2. Add 2 Button Blocks
3. Set one button to fill and one to outline

Video of how buttons should look and behave 

https://user-images.githubusercontent.com/7208249/195336236-f47b3ace-11c5-44a8-b6f1-ff8bf718e36a.mov

How to additionally verify the designs: 
1. Open inspect element and switch :hover, :active, :focus states and compare to the design

https://user-images.githubusercontent.com/7208249/195336569-27b4ad7b-4855-4731-be56-476ce7b61c79.mov
